modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/OperationMessageProcessor.java

```diff
@@ -307,9 +307,9 @@ protected ParameterValueResolver getParameterValueResolver() {
   public ProcessingType getProcessingType() {
     ProcessingType processingType = asProcessingType(operationModel.getExecutionType());
     if (processingType == CPU_LITE && !operationModel.isBlocking()) {
-      // If processing type is CPU_LITE and operation is non-blocking then use ASYNC processing type so that the Flow can return
+      // If processing type is CPU_LITE and operation is non-blocking then use CPU_LITE_ASYNC processing type so that the Flow can return
```

Comment:
how about operations that are not IO? for instance, if a router were an operation